addons/website/static/src/builder/plugins/list_group_option.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="website.ListGroupOption" >
+    <BuilderRow label.translate="Color">
+        <BuilderColorPicker enabledTabs="['solid', 'custom']" styleAction="'--s_list_group-icon-color'" />
+    </BuilderRow>
+    <BuilderRow label.translate="Background">
+        <BuilderColorPicker enabledTabs="['solid', 'custom', 'gradient']" styleAction="'--s_list_group-icon-bg'" />
+    </BuilderRow>
```

Comment:
To discuss with @lebl-odoo but I wonder if it would not look better with the two colorpickers on one line, like in the "Scroll Down Button" options :thinking: With tooltips to distinguish them ("Apply an icon color" and "Apply a background color") 
I would also therefore rename the option as "Bullet Colors" or "Icon Colors", to discuss :+1: